lib/modules/uscore_v3.1.0/profile_definitions/us_core_encounter_definitions.rb

```diff
@@ -6,6 +6,24 @@ class USCore310EncounterSequenceDefinitions
       MUST_SUPPORTS = {
         extensions: [],
         slices: [],
+        references: [
+          {
+            path: 'subject',
+            resource_types: [
+              'Patient'
+            ]
+          },
+          {
+            path: 'participant.individual',
+            resource_types: [
+              'Practitioner'
+            ]
+          },
+          {
+            path: 'location.location',
+            resource_types: []
```

Comment:
Should this be `Location`?

Reply:
No, the path is actually `location.location`, because the `location` element has a `location` field that can store the reference, as well as other fields.  See http://hl7.org/fhir/us/core/STU3.1/StructureDefinition-us-core-encounter.html (the Full View is more useful to see why there is that odd looking `location.location`.